lib/routes/chuapp/chuapp.ts

```diff
@@ -0,0 +1,148 @@
+import { Context } from 'hono';
+import { load } from 'cheerio';
+import { Data, Route, DataItem } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/:category?',
+    categories: ['game'],
+    example: '/chuapp/daily',
+    parameters: {
+        category: '栏目分类，见下表',
+    },
+    description: `
+    | \`category\` | 栏目分类 |
+    | ------------ | ------- |
+    | \`daily\`    | 每日聚焦 |
+    | \`pcz\`      | 最好玩   |
+    | \`night\`    | 触乐夜话 |
+    | \`news\`     | 动态资讯 |
+    `,
```

Comment:
Indent with either 0 or 2 spaces, otherwise the markdown content won't be rendered properly.

Reply:
Thanks for the review. Maybe we can also reflect this requirement in the guide? The current version does not mention it in either the Create Route section or the Script Standard section. (The code example hinted at it, though.)